hack/test/run_ut.sh

```diff
@@ -1 +1,28 @@
+#!/usr/bin/env bash
+  
+# Copyright 2022 Openwald Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OPENWALD_ROOT=$(cd -P "$(dirname "${BASH_SOURCE-$0}")/../../"; pwd -P)
+source "${OPENWALD_ROOT}/hack/lib/util.sh"
+
+# Run ut-test
+util::hello_openwald
+log::tips "Run ut-test."
+cd ${OPENWALD_ROOT}
```

Comment:
ignore

Reply:
> _ShellCheck:_ Double quote to prevent globbing and word splitting.
> 
> (at-me [in a reply](https://help.sonatype.com/lift/talking-to-lift) with `help` or `ignore`)
> 
> Was this a good recommendation? [ [🙁 Not relevant](https://www.sonatype.com/lift-comment-rating?comment=191935350&lift_comment_rating=1) ] - [ [😕 Won't fix](https://www.sonatype.com/lift-comment-rating?comment=191935350&lift_comment_rating=2) ] - [ [😑 Not critical, will fix](https://www.sonatype.com/lift-comment-rating?comment=191935350&lift_comment_rating=3) ] - [ [🙂 Critical, will fix](https://www.sonatype.com/lift-comment-rating?comment=191935350&lift_comment_rating=4) ] - [ [😊 Critical, fixing now](https://www.sonatype.com/lift-comment-rating?comment=191935350&lift_comment_rating=5) ]

ignore